testresults/src/org/labkey/testresults/view/runDetail.jsp

```diff
@@ -68,8 +68,7 @@
     <!--Header content, username run Id, and other information about the run-->
     <h2>
         <%=h(run.getUserName() + " : " + df.format(run.getPostTime()))%> &nbsp;
-        <img id="flagged" flagged="false"
-            src="<%=h(contextPath)%>/TestResults/img/<% if (run.isFlagged()) { %>flagon<% } else { %>flagoff<% } %>.png"
+        <img id="flagged" src="<%=h(contextPath)%>/TestResults/img/<% if (run.isFlagged()) { %>flagon<% } else { %>flagoff<% } %>.png"
```

Comment:
Maybe replace this with
src="<%=getWebappURL("/TestResults/img/" + (run.isFlagged() ? "flagon" : "flagoff" + ".png"))%>"